sunpy/time/time.py

```diff
@@ -159,119 +251,13 @@ def parse_time(time_string, time_format='', **kwargs):
     >>> sunpy.time.parse_time('2005-08-04T00:01:02.000Z')
     datetime.datetime(2005, 8, 4, 0, 1, 2)
     """
-    if time_format == 'datetime':
-        parse_time.dispatch(datetime)(time_string, time_format='', **kwargs)
-    elif time_format == 'utime':
-        parse_time.dispatch(float)(time_string, time_format='', **kwargs)
+    if time_format == 'utime':
+        return convert_time.dispatch(float)(time_string, **kwargs)
```

Comment:
instead of doing this it should be `convert_time(float(time_string), **kwargs)` because if it isn't convertable to a float it's going to error anyway.

Reply:
Okay, will do this!